neofetch

```diff
@@ -2178,6 +2178,32 @@ END
                          "${XDG_CONFIG_HOME}/sakura/sakura.conf")"
         ;;
 
+        "st")
+            if term_font="$(ps -o command= -p "$parent" | grep -E '\-f\s?.*?:?')"; then
```

Comment:
```sh
term_font="$(ps -o command= -p "$PPID" | grep -F -- "-f")"

if [[ "$term_font" ]]; then
```

Reply:
`$PPID` from what I can see is the parent of `neofetch` (the shell: `zsh` in my case). `$parent` is the terminal.